Assets/Mirror/Components/NetworkLobbyPlayer.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Mirror
+{
+    [DisallowMultipleComponent]
+    [AddComponentMenu("Network/NetworkLobbyPlayer")]
+    [HelpURL("https://vis2k.github.io/Mirror/Components/NetworkLobbyPlayer")]
+    public class NetworkLobbyPlayer : NetworkBehaviour
+    {
+        [SerializeField] public bool ShowLobbyGUI = true;
+
+        [SyncVar]
+        public bool ReadyToBegin = false;
+
+        [SyncVar]
+        public int Index;
+
+        /// <summary>
+        /// Do not use Start - Override OnStartrHost / OnStartClient instead!
+        /// </summary>
+        public void Start()
+        {
+            if (isClient) SceneManager.sceneLoaded += ClientLoadedScene;
+
+            if (NetworkManager.singleton as NetworkLobbyManager)
+            {
+                ReadyToBegin = false;
+                OnClientEnterLobby();
+            }
+            else
+                Debug.LogError("LobbyPlayer could not find a NetworkLobbyManager. The LobbyPlayer requires a NetworkLobbyManager object to function. Make sure that there is one in the scene.");
+        }
+
+        void OnDisable()
+        {
+            SceneManager.sceneLoaded -= ClientLoadedScene;
+        }
+
+        void ClientLoadedScene(Scene arg0, LoadSceneMode arg1)
+        {
+            NetworkLobbyManager lobby = NetworkManager.singleton as NetworkLobbyManager;
+            if (lobby)
+            {
+                // dont even try this in the startup scene
+                if (SceneManager.GetActiveScene().name == lobby.LobbyScene)
+                {
+                    Application.targetFrameRate = 10;
```

Comment:
why 10?

Reply:
Because there's no need to have the CPU screaming at 99% for a UI scene, and I think this predated you implementing targetFramerate management, or at least my being aware you had.